tilesets/scripts/cli.py

```diff
@@ -303,35 +302,41 @@ def validate_source(features):
 @cli.command("add-source")
 @click.argument("username", required=True, type=str)
 @click.argument("id", required=True, type=str)
-@cligj.features_in_arg
+@click.argument("filename", required=True, type=str)
```

Comment:
@Gerdie using `@cligj` means that we both:
1. Can handle ld and non ld geojson (iow abstract this for the user)
2. Validate and parse the geojson before passing in

I think we'll have to find a way to stream from input vs from a file. I think we have a few options:
1. Stream in to temp file, stream temp file to upload
2. `cligj` evaluates lazily so directly streaming in chunks (probably would be the weirdest to implement but most efficient overall)
3. ?